### PR TITLE
Feature/pet applications build out ln

### PR DESCRIPTION
**Pull request by anewb87**
### 1. What changed?
Two new components (ApplicationContainer and ApplicationCard) were built out, styled, and nested into the PetApplications component.

### 2. Why is this change necessary?
This change allows the user to see the peoples' information who have applied to adopt their pet. 

### 3. What changed technically that may impact others?
This was a build out of new components to be displayed in one that already had been created. It's all self-contained and should not, as yet, impact anyone else technically.

### 4. How do we test it?
Added Cypress testing to check all the elements are displayed on the page.

### CODE REVIEW

**Review by ectweitmann — APPROVED** [review]
Nice job Lexy! This is solid. Looking forward to hopefully transitioning to some real data tomorrow.

File: src/Components/ApplicationCard.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  return (
+    <section className='application-card'>
+      <div className='applicant-name'>
+        <h3>Loki Lucifer</h3>
+        <p className='email'>catsmeow@gmail.com</p>
+      </div>
+      <div className='applicant-details'>
+        <p className='intro-label'>Applicant Introduction:</p>
+        <p className='intro'>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum. tetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
```
Comment (Fordo29) [inline src/Components/ApplicationCard.js:13]: Thanks for the quick action on some filler dummy data.

Comment (StevieAmb) [inline src/Components/ApplicationCard.js:13]: Nice job checking for edge casing with this form, and checking for the different lengths of the inputs, and helping write the CSS around it to make it accessible. Great forward thinking,

File: src/Styles/ApplicationCard.scss [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  width: 75%;
+  height: 200px;
+  box-shadow: 8px 8px 9px grey;
+  border-radius: 10px;
+}
+
+.applicant-name {
+  font-size: xx-large;
+  max-width: 25%;
+  overflow-wrap: break-word;
```
Comment (Fordo29) [inline src/Styles/ApplicationCard.scss:18]: Nice find on how to wrap a name or email mid-word

File: cypress/integration/pet_applications_spec.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
+describe('Pet Applications Component', () => {
```
Comment (StevieAmb) [inline cypress/integration/pet_applications_spec.js:1]: Great job writing these tests before merging to see whether it is giving us what we want it to.

File: src/Styles/ApplicationCard.scss [hunk 4]
```diff
@@ -0,0 +1,42 @@
+@import '_variables.scss';
+
+.application-card {
```
Comment (StevieAmb) [inline src/Styles/ApplicationCard.scss:3]: I love this box shadow.

**Review by StevieAmb — APPROVED** [review]

**Review by Fordo29 — APPROVED** [review]
Excited for tomorrow to really dive into the logic

**Pull request merged**